assets/scripts/admin-opt-out.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+(function ($) {
+  var $dest = $('#core-standards-optout-script');
+  // Restores the default opt-out script if textarea is empty.
+  if (!$dest.val().trim().length) {
+    load_default_script($dest);
```

Comment:
The Core Standards plugin is about sane & smart standards… — Why would I ever want to override the opt-out script with a custom one, editing in a textarea?  Wouldn't we improve the default script instead?

Was there a particular request or requirement for making it possible to override the default script?

Reply:
@sun Although I don't really like the idea, this is defined as a "nice to have" requirement in the task.

Reply:
I do not like the idea either and would recommend to remove it.  The only configuration I expect from this feature are the GA IDs.

Reply:
@sun Added a comment to the task recommending Peter about discarding this feature.